connectivity-exporter/internal/tools/tools.go

```diff
@@ -5,11 +5,11 @@
 // Do not build this file by default, it's only parsed by go mod to
 // pull the dependencies.
 
+//go:build tools
 // +build tools
```

Comment:
What's the difference between `+build` and `go:build`?

Reply:
Starting with go1.17 it is preferred over `+build` 
https://golang.org/doc/go1.17

Reply:
Here are the referenced docs: https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md

Reply:
Note: together with @wyb1 we found out that `gofmt` that is shipped with `go 1.17` is suggesting these changes.
This is a good hint for upgrading Go from 1.16 to 1.17.